### DISCUSSION TITLE
Mobile App ?

### DISCUSSION THREAD

**Discussion by Hamza-Aouni** [dailydotdev/daily#233] [Feature Requests]
Hey Daily dev :D , i guess it will be a brilliant idea if it would be a mobile app, especially nowadays, let's say 80% of people use mobile phones a lot in all add to the web extensions.

**Comment by esau-morais** [comment]
If you are updated, there's a Progressive Web App created by Daily.dev (called Daily.go). But the upcoming web version of Daily.dev (current in its beta version) can be an excellent opportunity to improve and even replace the old Daily go.

**Note:** _You can see a discussion simular [here](https://github.com/dailydotdev/daily/discussions/231)_

### My opinion

I would probably say no for native apps in the Daily.dev case. A great advantage of PWAs over native apps is that they're independent of app stores. Since 25% of smartphone users delete apps due to lack of storage space, this independence can be a great advantage. Customers don't need to download PWAs and use lots of space. All that's required is a web browser.

Now, let's see what the maintainers say about it :) Hope you understand my point.

**Reply by Hamza-Aouni** [reply]
Indeed, I do respect what you've said but we are talking about 2021 "25% of smartphone users delete apps due to lack of storage space " this was in the past, "People Spent 90% of Their Mobile Time Using Apps in 2019", by the way, I am one of the persons who hates mobile phones and I don't spend a lot of time in it. but the idea comes to my mind when I see every day all people are using their phones even if they do have their laptops with them.

**Reply by esau-morais** [reply]
> Indeed, I do respect what you've said but we are talking about 2021 "25% of smartphone users delete apps due to lack of storage space " this was in the past, "People Spent 90% of Their Mobile Time Using Apps in 2019", by the way, I am one of the persons who hates mobile phones and I don't spend a lot of time in it. but the idea comes to my mind when I see every day all people are using their phones even if they do have their laptops with them.

I got what you mean, but there are more reasons why Daily.dev wouldn't be a good match for native apps:

1. User preference (it's not just about space)

![50% of smartphone users are more likely to use  a company or brand's mobile site  when browsing or shopping on a smartphone because they don't want to download an app](https://apiko.com/blog/content/images/size/w1500/2020/07/0jLaB-data-smartphone-user-mobile-shopping-preferences-download.jpg)

2. Too expensive

Believe me, native apps are not that easy to build or even maintain. Besides the need to build two versions of the native app

---

### On the other side... PWA is:

1. Adaptive
2. Optimized performance
3. Optional installation
4. No obligatory updates
5. Linkable
6. Secure

**Note:** _This is my sincere opinion_

**Comment by idoshamun** [comment]
Hi @Hamza-Aouni,

Thanks for sharing your idea with us :)
Expanding to mobile users is a great opportunity for daily.dev as both you and @esau-morais say.
I agree with @esau-morais that a PWA is a great fit for us for a number of reasons.
The critical reason is that we're a very small team and cannot afford to maintain and build so many platforms.
I think PWA will be a great solution for everyone who doesn't like the extension or would like to access the content through their mobile phone.

**Comment by nimrodkra** [comment]
Quick update on this thread: we released a beta version of daily.dev web app. It means daily.dev is now available as a PWA for mobile as well ✨ Feel free to try it here: https://app.daily.dev

cc: @Hamza-Aouni @esau-morais let us know what you think :)

**Reply by Hamza-Aouni** [reply]
Hey !  Very nice release, i found it awsome, smooth and very responsive well done team daily dev

**Comment by nimrodkra** [comment]
Another update! Our PWA is also available as an Android app: https://play.google.com/store/apps/details?id=dev.daily

**Comment by vinagroup** [comment]
10 N+ Android? I think it good.
With app native, we can have rating, feedback and more...!
Waiting for version IOS appstore.

**Comment by perry2Erip** [comment]
One of the big problems with the current PWA implementation on Android a tleast is that Chrome needs to be enabled. I use DuckDuckGo on my Android device and using the PWA just opens a new tab on the DuckDuckGo app. I know that's a DuckDuckGo problem as well but it's really limiting to depend on Chrome.
